### ISSUE TITLE
close buttons a11y

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/react-invenio-deposit#540]
- The close-button on the dates field in the deposit form is missing aria-label.
- Should have `aria-label={i18next.t("Close")}`
- Check also the other close buttons in the form 
- Check that the icon within the buttons are hidden (`aria-hidden="true"`)

![Screenshot 2022-07-01 at 13 57 19](https://user-images.githubusercontent.com/21052053/176890477-430060fc-f5c3-490c-95e0-61003d4bfabc.png)
